component/core/src/main/java/org/exoplatform/social/core/jpa/storage/entity/MetadataPropertiesEntity.java

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
the new entity is not needed to do the Inner join query, you can check the query with the actual HashMap ?